cssdisplay.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>CSSDisplay</title>
+    <link rel="stylesheet" href="cssdisplay.css">
+</head>
+
+<body>
+    <ul class="navigation-bar">
+        <img class="logo" src="images/logo.jpg">
```

Comment:
img trebuie pus tot intr-un **li** . UL-ul accepta ca si copii directi doar li-uri
Astfel, ori scoti img in afara ul-ului ori pui img intr-un li